pkg/controller/atlascluster/atlascluster_controller.go

```diff
@@ -243,11 +356,24 @@ func (r *AtlasClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return err
 	}
 
+	// Watch for Backup schedules
+	err = c.Watch(&source.Kind{Type: &mdbv1.AtlasBackupSchedule{}}, watch.NewBackupScheduleHandler(r.WatchedResources))
+	if err != nil {
+		return err
+	}
+
+	// Watch for Backup policies
+	err = c.Watch(&source.Kind{Type: &mdbv1.AtlasBackupPolicy{}}, watch.NewBackupPolicyHandler(r.WatchedResources))
+	if err != nil {
+		return err
+	}
+
 	return nil
 }
 
 // Delete implements a handler for the Delete event.
 func (r *AtlasClusterReconciler) Delete(e event.DeleteEvent) error {
+	// TODO: Add deletion for AtlasBackupSchedule and AtlasBackupPolicy
```

Comment:
are there tickets to track this?

Reply:
Forgot to remove that. The deletion of schedule/policy will trigger reconciliation of clusters this schedule/policy is connected to. No need to add a separate deletion handler.